lib/src/source/git.dart

```diff
@@ -142,6 +142,8 @@ class GitSource extends Source {
     return {'relative': relative, 'url': url};
   }
 
+  /// Returns [path] normalized.
```

Comment:
I'd document it as `/// Normalizes [path].` And name it `_normalizePath`, and say that [path] must be a string or `null`.